infrastructure/azure.tf

```diff
@@ -425,8 +417,6 @@ locals {
   azurerm_apim_eventhub_name                = "${var.azurerm_resource_name_prefix}-apim-eventhub-${var.environment_short}"
   azurerm_notification_hub                  = "${var.azurerm_resource_name_prefix}-notificationhub-${var.environment_short}"
   azurerm_notification_hub_ns               = "${var.azurerm_resource_name_prefix}-notificationhubns-${var.environment_short}"
-  azurerm_kubernetes_name                   = "${var.azurerm_resource_name_prefix}-k8s-${var.environment_short}"
```

Comment:
We should use the "aks_cluster" or "kubernetes" label in a consistent way, need to chose one (aks_cluster?) and replace it in locals names.

Reply:
Let me move everything back to kubernetes. I want to be consistent as much as possible with the azurerm names.